py/server/deephaven/table.py

```diff
@@ -1703,15 +1938,15 @@ def slice(self, start: int, stop: int) -> Table:
         """Extracts a subset of a table by row positions into a new Table.
 
           If both the start and the stop are positive, then both are counted from the beginning of the table.
-          The start is inclusive, and the stop is exclusive. slice(0, N) is equivalent to :meth:`~Table.head(N)`
+          The start is inclusive, and the stop is exclusive. slice(0, N) is equivalent to :meth:`~Table.head` (N)
```

Comment:
Why is it given as `~Table.head` (N) and not `~Table.head(N)`?

Reply:
because Sphinx doesn't like the format